src/react-sortable-tree.js

```diff
@@ -634,6 +636,9 @@ class ReactSortableTree extends Component {
           {({ height, width }) => (
             <ScrollZoneVirtualList
               {...scrollToInfo}
+    				  ref={el => {
+                this.scrollZoneVirtualListComponent = el
```

Comment:
https://github.com/fritz-c/react-sortable-tree/blob/master/src/react-sortable-tree.js#L98

The element is already defined.